docker/repo.sls

```diff
@@ -13,7 +16,7 @@
 docker-package-repository:
   pkgrepo.{{ repo_state }}:
     - humanname: {{ grains["os"] }} {{ grains["oscodename"]|capitalize }} {{ humanname_old }}Docker Package Repository
-    - name: deb {{ url }}
+    - name: deb [arch={{ grains["osarch"] }}] {{ url }}
```

Comment:
Is there any open issue this fixes?

Reply:
**repo.sls**: No open issues ... probably people choose to manually change the sources.list file to include [arch=amd64] because in all my deployments I had to. 

**init.sls**: Since docker.config is already an object, I do not expect it needs any filters. But then again I could be mistaken.

Reply:
The url that I get is:  deb [arch=amd64] https://download.docker.com/linux/ubuntu bionic stable ... the one that gave problems is  apt.dockerproject.org.

So this seems to be fixed already, but not by me ;).